### ISSUE TITLE
ARIA required context role (ff89c9) Fail 3 ill-defined

### ISSUE THREAD

**Issue by tombrunet** [act-rules/act-rules.github.io#1703]
This is regarding Fail example 3 for rule ff89c9:
https://act-rules.github.io/rules/ff89c9
```
<div role="list">
	<div aria-label="menu">
		<div role="listitem">List item 1</div>
		<div role="listitem">List item 2</div>
	</div>
</div>
```

The `div` in this case implicitly maps to the `generic` role. Per ARIA 1.2, `aria-label` is prohibited on generic (https://www.w3.org/TR/wai-aria-1.2/#generic). Therefore, its inclusion as a global attribute is questionable in this case. It feels like that attribute should be ignored since it's prohibited (even though Chrome isn't ignoring it). If you don't want to wade into that ambiguity, it may be better to replace the `aria-label` with `aria-describedby`. If you want to stick with `aria-label`, a more nuanced explanation may be needed.

**Comment by WilcoFiers** [comment]
@tombrunet Thanks for raising this. Right now, neither HTML-AAM nor HTML in ARIA says that `div` elements have an implicit role of `generic`. Either way you have a fair point that at some point failed example 3 may be better off with a different global ARIA attribute. I talked to Carlos and we thought `aria-busy` might work. Would you be open to create a PR for this?

**Comment by tombrunet** [comment]
I think both haven't been fully updated for ARIA 1.2. ARIA 1.2 notes it here:
https://www.w3.org/TR/wai-aria-1.2/#generic
> The generic role is intended for use as the implicit role of generic elements in host languages (such as HTML div or span)

Since it has no semantics, it may not show in AAM. It does make me question if this is really a failure since the div has no semantics regardless of whether valid ARIA attributes are specified. Looks like the PR is in #1700 but I added a comment since I think the intended change was misunderstood.

**Comment by ajanec01** [comment]
Hi @tombrunet. According to the expectation of the rule, each list item must be a **child** of an element with `role="list"`. In this respect [ARIA context role definition](https://www.w3.org/TR/wai-aria-1.2/#scope) is less strict as it refers to the definition of [owned element](https://www.w3.org/TR/wai-aria-1.2/#dfn-owned-element) which would indeed make that failed example a pass example. 

When using VO with Safari, this failed example is not read out as a list of two items. Instead `<div aria-label="menu">` is treated as a grouping container.

As for #1700, yes, lapse of concentration on my part.

**Comment by tombrunet** [comment]
Simply because Safari implemented it wrong doesn't mean that it's correct. I haven't seen anything that indicates that a div should implicitly be a group for any reason.

**Comment by WilcoFiers** [comment]
I reached out to Scott O'Hara (editor of html-aam) about this when you brought this up. They aren't going to make just every div a `generic` element that needs to be included in the accessibility tree. We'll have to see exactly how this ends up being defined. The work just hasn't been done yet.

Regardless of that, the fix I suggested earlier, to use `aria-busy=false` would work though. Alternatively, `aria-live=polite` would do the trick too.  I'll open a PR for this.

**Comment by tombrunet** [comment]
@WilcoFiers https://github.com/act-rules/act-rules.github.io/pull/1700/files had the aria-busy change in it, which is why I didn't create my own PR - though I guess it's draft.

I think the `aria-busy` fix is more understandable for now, however, my argument may go beyond ACT. I have a hard time arguing that attributes on a node with no semantics should have any effect - either way, it has no role, so having it affect role hierarchy is confusing to me. It feels like this should be a pass, rather than a fail. But, I'm not sure if that oddity is an issue for ARIA, or ACT.
